app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -105,7 +105,20 @@ class PlanWizardInstancePropertiesStepTable extends React.Component {
       const clusterId = additionalData.rowData.ems_cluster_id;
       const tenantId = destinationTenantIdsBySourceClusterId[clusterId];
       const tenant = tenantId && tenantsWithAttributesById[tenantId];
-      const options = tenant ? tenant[optionsAttribute] : [];
+      const allFitForVM = () => {
```

Comment:
It might be nice to extract this method to a helpers file

Reply:
I have extracted this method in a helper and have addressed the above `[].find()/undefined` concern there